src/ontology/uberon-edit.obo

```diff
@@ -224701,6 +224701,15 @@ intersection_of: connects UBERON:7500034 ! distal-most point of trochlea of hume
 intersection_of: connects UBERON:7500048 ! proximal-most point of ventral tubercle of humerus
 created_by: https://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500061
+name: trochlear ridge of humerus
+def: "The ridge of the trochlea on a humerus lies in the middle of the distal end of a humerus on the lateral side of the trochlea and the medial side of the capitulum (capitellum)." [ORCID: 0000-0003-2699-3066]
+intersection_of: UBERON:0004765 ! skeletal element
+intersection_of: part_of UBERON:0000144 ! trochlea of humerus
```

Comment:
See https://github.com/obophenotype/uberon/pull/1897/files#r676113805 for suggested change to classification.

Reply:
I think you said you'd done this in a comment on #1897 (?)Looks like this change still needs to be pushed - or is it on the other PR?

Reply:
done.